### DISCUSSION TITLE
Ensemble of Local and Global models

### DISCUSSION THREAD

**Discussion by kostasek** [alan-turing-institute/sktime#5571] [Q&A]
Is there a way to combine forecasts from local and global methods?

**Comment by fkiraly** [comment]
yes, or of any methods - use one of the ensembles:

* `ColumnEnsembleForecaster` for using different forecasters on different variables
* `HierarchyEnsembleForecaster` for using different forecasters on different hierarchy nodes
* `Detrender` (a transformer) to have one forecaster forecast trend, the other one residuals
* one of the ensemble forecasters for traditional ensembling: https://www.sktime.net/en/stable/api_reference/forecasting.html#ensembles-and-stacking

**Reply by fkiraly** [reply]
forgot, you can also do signal decomposition, forecast individual components differently, then recompose, this is a more general case of `Detrender` - exampels are `VmdTransformer`, or `STLTransformer`

**Comment by kostasek** [comment]
Thanks Franz, but I am not sure if I set the issue correctly. What I meant previously was if there is an easy way to create an ensemble forecaster that comprises of a global model that is trained to forecast just a single time series of interest and a local model eg. an ARIMA model of that one single time series...
